packages/riipen-ui/src/components/RadioButton.test.jsx

```diff
@@ -0,0 +1,158 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import { faFlag } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import RadioButton from "./RadioButton";
+
+const icon = i => <FontAwesomeIcon icon={i} />;
+
+describe("<RadioButton>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<RadioButton />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<RadioButton />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("checked prop", () => {
+    it("sets element's class name or prop value when checked is true", () => {
+      const checked = true;
+
+      const wrapper = mount(<RadioButton checked={checked} readOnly />);
+
+      expect(
+        wrapper
+          .find("RadioButton")
+          .childAt(0)
+          .hasClass("checked")
+      ).toEqual(true);
+      expect(wrapper.find("label").hasClass("checked")).toEqual(true);
+      expect(wrapper.find("input").props().checked).toEqual(checked);
+      expect(wrapper.find("Typography").props().color).toEqual("initial");
+    });
```

Comment:
I am actually wondering if this should be 4 separate tests. The test name no longer reflects everything that is being tested here. 

As a rule of thumb, it's generally a good idea to only actually test 1 thing per test. Reason being, if this test fails the test name and what causes it to fail now no longer align and it can be hard to debug. Also, the first expect might fail causing the others to actually not run. So you go and fix the first expect, but then the 2nd fails, and so on and so on. So you actually had only 1 test failure, but more than 1 thing was failing.